packages/orbit-tailwind-preset/src/presets/__tests__/configs.test.ts

```diff
@@ -4,7 +4,7 @@ import orbitPreset from "../..";
 
 describe("orbitPreset", () => {
   it("should match snapshot", () => {
-    const cfg = resolveConfig(orbitPreset());
+    const cfg = resolveConfig(orbitPreset({ disablePreflight: true }));
     expect(cfg).toMatchSnapshot();
     expect(cfg.corePlugins).not.toContain("preflight");
```

Comment:
This test only checks if the config value is present, doesn't it? Can we check the value when is present (test below)?

Reply:
it's checking opposite, if it's not present, sorry, not sure if I understand what you mean

Reply:
Sorry, my bad. I didn't realize `resolveConfig` was present. I see that it converts it into an array and only has `preflight` when is enabled. Looks good then! Resolving